addons_marketplace/marketplace-manager/api/v1/marketplace_blueprint.py

```diff
@@ -0,0 +1,34 @@
+from db import marketplace_db
+from flask.views import MethodView
+from flask_smorest import Blueprint, abort
+from services import marketplace_service
+
+from .schema import MarketplaceAddonSchema, MarketplaceFilterSchema
+
+marketplaceblp = Blueprint("Marketplace Api", "marketplace_api", url_prefix="/api/v1/marketplace")
+
+
+@marketplaceblp.route("/")
+class MarketplaceAddonsController(MethodView):
+    @marketplaceblp.arguments(MarketplaceFilterSchema, location="query")
+    @marketplaceblp.response(
+        200, MarketplaceAddonSchema(many=True), content_type="application/json"
+    )
+    def get(self, query={}):
+        return list(marketplace_db.find_approved_addons(query))
+
+    @marketplaceblp.arguments(MarketplaceAddonSchema, location="json")
+    @marketplaceblp.response(201, MarketplaceAddonSchema, content_type="application/json")
+    def post(self, data):
```

Comment:
Is this method intended to add/post multiple addons at once or only single ones? If only single once then I think this method would be better placed at the singular "MarketPlaceAddonController" below not at this plural one.

Reply:
Ah yes I see. Ok makes sense